### PR TITLE
CTCP-516: Initial commit using p5 service guide as skeleton

### PR DESCRIPTION
**Pull request by drnaylor-oc**

### CODE REVIEW

**Review by liandra-giacomine — COMMENTED** [review]
I think we are missing .ruby-version and .node-version

**Comment by drnaylor-oc** [comment]
Good spot - forgot to copy hidden files across - added!

**Review by liandra-giacomine — APPROVED** [review]

**Comment by drnaylor-oc** [comment]
Going to hold off merging until the orchestrator job has been merged - then this will deploy and will release Peter to do his work.

**Pull request merged**
